packages/ui-patterns/form/FormLayout.tsx

```diff
@@ -0,0 +1,356 @@
+import { FormItemLayout } from './FormItemLayout/FormItemLayout'
+
+const items = [
+  {
+    id: 'recents',
+    label: 'Recents',
+  },
+  {
+    id: 'home',
+    label: 'Home',
+  },
+  {
+    id: 'applications',
+    label: 'Applications',
+  },
+  {
+    id: 'desktop',
+    label: 'Desktop',
+  },
+  {
+    id: 'downloads',
+    label: 'Downloads',
+  },
+  {
+    id: 'documents',
+    label: 'Documents',
+  },
+] as const
+
+export const Page = () => {
+  const FormSchema = z.object({
+    username: z.string().min(2, {
+      message: 'Username must be at least 2 characters.',
+    }),
+    kevins_input: z.string().min(6, {
+      message: 'Username must be at least 6 characters.',
+    }),
+    email: z
+      .string({
+        required_error: 'Please select an email to display.',
+      })
+      .email(),
+    consistent_settings: z.boolean().default(false).optional(),
+    switch_option: z.boolean().default(false).optional(),
+    items: z.array(z.string()).refine((value) => value.some((item) => item), {
+      message: 'You have to select at least one item.',
+    }),
+    type: z.enum(['all', 'mentions', 'none'], {
+      required_error: 'You need to select a notification type.',
+    }),
+  })
+
+  const form = useForm<z.infer<typeof FormSchema>>({
+    resolver: zodResolver(FormSchema),
+    defaultValues: {
+      username: '',
+      items: ['recents', 'home'],
+    },
+  })
+
+  function onSubmit(data: z.infer<typeof FormSchema>) {
+    console.log(data)
+  }
+
+  const UserIcon = () => {
+    return (
+      <div>
+        <img
+          className="relative h-4 w-4 rounded-full"
+          src="https://avatars.githubusercontent.com/u/8291514?v=4"
+        />
+      </div>
+    )
+  }
+
+  return (
+    <Form_Shadcn_ {...form}>
+      <form onSubmit={form.handleSubmit(onSubmit)} className="w-[420px] flex flex-col gap-8">
+        <div>
+          <h1 className="text-foreground">Welcome</h1>
+          <p className="text-foreground-light">Please fill in the following</p>
+        </div>
+        <div role="separator" className="h-px bg-border-muted w-full" />
+
+        <FormField_Shadcn_
+          control={form.control}
+          name="username"
+          render={({ field }) => (
+            <FormItemLayout
+              label="Function name"
+              description="This is your public display name."
+              layout="horizontal"
+              labelOptional="Optional"
+              afterLabel={<InfoTooltip side="right">You can also rename this later.</InfoTooltip>}
+            >
+              <FormControl_Shadcn_>
+                <Input
+                  icon={<Box strokeWidth={1.5} size={16} />}
+                  placeholder="mildtomato"
+                  {...field}
+                />
+              </FormControl_Shadcn_>
+            </FormItemLayout>
+          )}
+        />
+
+        <div role="separator" className="h-px bg-border-muted w-full" />
```

Comment:
just to check as well - we do have a Separator component right?

Reply:
Not a global one, think i need tom add one. 
don't worry about it, this is an example page that will keep getting updated

Reply:
oh sorry we do

Reply:
fixed